austrakka/components/project/__init__.py

```diff
@@ -76,22 +76,21 @@ def project_update(
 
 @project.command('set-dashboard', hidden=hide_admin_cmds())
 @opt_name(help="name of a known dashboard")
-@click.argument('project-id', type=int)
-def dashboard_set(project_id: int, name: str):
+@click.argument('project-abbreviation', type=str)
+def dashboard_set(project_abbreviation: str, name: str):
```

Comment:
Should we use `abbrev`? I think we're using that shorthand elsewhere

Reply:
I think it should be project abbrev rather than abbrev - we have both in various places, and the ones that have abbrev are a little confusing IMO. But project-abbreviation probably doesn't match ANY existing convention, I'll change it

Reply:
Looking, you're right, abbrev is everywhere - for plots, proformas, projects. project-abbreviation is used in set-dashboard, which is probably where I got it